.github/workflows/submitty_ci.yml

```diff
@@ -520,3 +520,14 @@ jobs:
       - uses: ./.github/actions/e2e-Setup-Composite
       - name: Run integration tests
         run: sudo -E env "PATH=$PATH" python3 /usr/local/submitty/test_suite/integrationTests/run.py
+
+  Notify-Failure:
+    runs-on: ubuntu-latest
+    if: ${{ always() && contains(join(needs.*.result, ','), 'failure') && github.event.pull_request.title }}
```

Comment:
Do you need something about only the main branch here?  It might be useful to put a few comments describing the logic here in case someone stumbles into this in the future.

Reply:
Yes, that got lost with my testing earlier... I will add comments too.